Vox scars

TrixxedHeart opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Gives Vox species scar parity, which means they now have all the scars.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Vox don't have em, or really any markings really

## Technical details
Adds vox_scars.rsi + Edited scar sprites to fit Vox better.
Changes vox_tattoos.ftl to vox.ftl to match the ftl naming scheme of other species.

## Media
![image](https://github.com/user-attachments/assets/b8842ac1-2346-4413-9da6-b6dedc34007a)
![image](https://github.com/user-attachments/assets/69edbeb9-3597-4d0e-9d85-4ed56b706009)
![image](https://github.com/user-attachments/assets/90901fce-3d42-42ac-a5bc-9e8a0193b8ba)
![voxscarsshowcase](https://github.com/user-attachments/assets/310d7e7b-2726-41bd-8802-f2f1154b7a5e)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
This renames the vox_tattoos.ftl locale to just vox.ftl to match naming schemes of other species!

**Changelog**
:cl:
- add: Vox now have scar markings.

TiniestShark commented:
Looks fine for the most part, main issues are: Face scar large looks strange in how high it comes up on the front view, but it not being anywhere on the face on the side view.
![scars 1](https://github.com/user-attachments/assets/d3cd75e5-4397-4415-9c60-49fa51f5e82c)

I'd also ask to take the top scars out. Vox don't have a biological sex so it doesn't make much sense for them to have them (Arachnids weren't given them either in the recent scar PR for this reason too btw.) and their idea of gender expression would probably be quite a bit different than ours.

TrixxedHeart commented:
> Looks fine for the most part, main issues are: Face scar large looks strange in how high it comes up on the front view, but it not being anywhere on the face on the side view. ![scars 1](https://private-user-images.githubusercontent.com/176858593/459636765-d3cd75e5-4397-4415-9c60-49fa51f5e82c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA5OTEwMjcsIm5iZiI6MTc1MDk5MDcyNywicGF0aCI6Ii8xNzY4NTg1OTMvNDU5NjM2NzY1LWQzY2Q3NWU1LTQzOTctNDQxNS05YzYwLTQ5ZmE1MWY1ZTgyYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYyN1QwMjE4NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04M2QzY2YyZDlmNzgyODkyOTQ3MDYzMWU3OGIyZDk1YmNiNzIyY2Q1OThiNDE5NDQ1MDJkNTdkZjQyMTgzMzY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cg3w4HcibtESRyuUpyjcCdI8M94a_GTVVIRLWzBha2Q)
> 
> I'd also ask to take the top scars out. Vox don't have a biological sex so it doesn't make much sense for them to have them (Arachnids weren't given them either in the recent scar PR for this reason too btw.) and their idea of gender expression would probably be quite a bit different than ours.

![image](https://github.com/user-attachments/assets/84e2b6e6-5396-4de2-aa50-1d54c6c929f7)
Changed the top surgery scars to have more generalized surgical names (I'll add more surgical scars in a different PR)

![image](https://github.com/user-attachments/assets/7b58c1ef-4566-43c3-9dc2-de5e93a7d9d6)
Adjusted the face scars to look less awkward from the side

TrixxedHeart commented:
Made fixes to the formatting I noticed some issues with
